BorrowMyProducts/pom.xml

```diff
@@ -57,7 +63,12 @@
 			<artifactId>spring-security-test</artifactId>
 			<scope>test</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>junit</groupId>
```

Comment:
Junit is already part of Spring

Reply:
Yeah I haven't noticed it.I think eclipse automatically added this dependency.I will remove it and update.

Reply:
updated it